cmd/site-controller/deploy-watch-all-ns.yaml

```diff
@@ -135,3 +135,11 @@ spec:
       containers:
       - name: site-controller
         image: quay.io/skupper/site-controller:master
+        securityContext:
+          capabilities:
+            drop:
+            - ALL
+          runAsNonRoot: true
+          allowPrivilegeEscalation: false
+          seccompProfile:
+            type: RuntimeDefault
```

Comment:
The `seccompProfile` field does not work with OKD 3.11.
All other fields seem to be working just fine.

Do you think it is better to keep it as is and just drop a note
somewhere explaining to remove it if running on 3.11 ?

Reply:
Lean towards keeping as is and will add a comment to indicate version considerations